src/validation.test.ts

```diff
@@ -0,0 +1,19 @@
+import { validateCheck } from 'validation';
+import { Check } from 'types';
+
+describe('validation test', () => {
+  it('should test target', () => {
+    const check: Check = {
```

Comment:
@mem -- can you add more tests here?  that will help whoever has to look at this next :)

Reply:
I added a bunch, thanks for bootstrapping! :smile: